rubicon_ml/client/mixin.py

```diff
@@ -326,17 +333,61 @@ def dataframes(self, tags=[], qtype="or"):
             The dataframes previously logged to this client object.
         """
         project_name, experiment_id = self._get_parent_identifiers()
-        dataframes = [
-            client.Dataframe(d, self)
-            for d in self.repository.get_dataframes_metadata(
-                project_name, experiment_id=experiment_id
-            )
-        ]
-
+        if name is not None:
+            dataframes = [
+                client.Dataframe(d, self)
+                for d in self.repository.get_dataframes_metadata(
+                    project_name, experiment_id=experiment_id
+                )
+                if d.name == name
+            ]
+        else:
+            dataframes = [
+                client.Dataframe(d, self)
+                for d in self.repository.get_dataframes_metadata(
+                    project_name, experiment_id=experiment_id
+                )
+            ]
         self._filter_dataframes(dataframes, tags, qtype)
 
         return self._dataframes
 
+    def dataframe(self, name=None, id=None):
+        """
+        Get the dataframe logged to this client object.
+
+        Parameters
```

Comment:
These params say artifact instead of dataframe.

Reply:
done thanks !